apps/download-service/src/app/app.module.ts

```diff
@@ -48,9 +59,12 @@ import {
         IdsClientConfig,
         XRoadConfig,
         VehiclesClientConfig,
+        DownloadServiceConfig,
```

Comment:
Download service should not need DownloadServiceConfig (used to create URLs to the download service).